apps/server/src/shared/domain/domainobject/board/file-element.do.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { createMock, DeepMocked } from '@golevelup/ts-jest';
+import { cardFactory, fileElementFactory } from '@shared/testing';
+import { FileElement } from './file-element.do';
+import { BoardNodeBuilder } from './types';
+
+describe(FileElement.name, () => {
+	const setup = () => {
+		const element = fileElementFactory.build();
+		const card = cardFactory.build();
+		const builder: DeepMocked<BoardNodeBuilder> = createMock<BoardNodeBuilder>();
+
+		return { element, builder, card };
+	};
+
+	describe('when trying to add a child to a file element', () => {
+		it('should call the specific builder method', () => {
+			const { element, builder, card } = setup();
+
+			element.useBoardNodeBuilder(builder, card);
+
+			expect(builder.buildFileElementNode).toHaveBeenCalledWith(element, card);
+		});
+
+		it('should throw an error ', () => {
```

Comment:
There should be another describe block here. Something like 'when trying to add an ivalid element'